src/ElasticAppSearchClient.php

```diff
@@ -28,7 +30,7 @@ class ElasticAppSearchClient implements ElasticAppsearchClientInterface {
    *
    * @var \GuzzleHttp\Client
    */
-  protected $client;
+  public $client;
```

Comment:
Any specific reason to make this guzzle client the public instead of protected? Also you already have a Guzzle client as `protected $httpClient;` at line 26.

Reply:
will get rid of guzzle client

Reply:
This is stil pending.